packages/sirius-web/frontend/sirius-web-application/src/index.ts

```diff
@@ -106,3 +106,6 @@ export {
   type ProjectSettingTabProps,
 } from './views/project-settings/ProjectSettingsView.types';
 export * from './views/project-settings/ProjectSettingsViewExtensionPoints';
+export { NewObjectModal } from './modals/new-object/NewObjectModal';
```

Comment:
I think you would need to export the props type too

Reply:
done